content/en/docs/contribute/style/style-guide.md

```diff
@@ -38,6 +38,92 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 {{< comment >}}[If you're localizing this page, you can omit the point about US English.]{{< /comment >}}
 
+## Guidelines for contributing content
+
+The Kubernetes documentation comprises the content of the 
+[kubernetes/website](https://github.com/kubernetes/website) source repository. 
+Located in the `kubernetes/website/content/<language_code>/docs` folder, the 
+majority of the Kubernetes documentation is specific to the [Kubernetes 
+project](https://github.com/kubernetes/kubernetes). The Kubernetes 
+documentation may also include content from projects in the 
+[kubernetes](https://github.com/kubernetes) and 
+[kubernetes-sigs](https://github.com/kubernetes-sigs) GitHub organizations if 
+those projects do not have their own documentation. Linking to active kubernetes, 
+kubernetes-sigs, and CNCF projects from the Kubernetes documentation is always 
```

Comment:
```suggestion
kubernetes-sigs, and {{< glossary_tooltip text="CNCF" term_id="cncf" >}} projects from the Kubernetes documentation is always
```
?

Reply:
thanks @stim for pointing out the glossary option! I moved the content guidelines to a new page as @kbhawkey suggested and added your tool tip to the text.